region/norfair/east/Cathedral Entrance.json

```
@@ -1217,6 +1253,32 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [4, 4],
+      "name": "R-Mode Spark Interrupt",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        {"or": [
+          "h_heatedCrystalFlashForReserveEnergy",
+          {"and": [
+            "h_heatProof",
+            "h_RModeCanRefillReserves",
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 40}},
+            {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 0}]}
```

I still think lower reqs > ending with a bit more energy
```suggestion
            {"partialRefill": {"type": "ReserveEnergy", "limit": 20}}
```